gui/default/syncthing/core/syncthingController.js

```diff
@@ -651,6 +652,17 @@ angular.module('syncthing.core')
             $scope.remoteNeedDevice = undefined;
         }
 
+        function setDefaultTheme(){
```

Comment:
Space after closing brace

Reply:
Do you use any JS linter/formatter? You suggest using `go fmt` and `go lint` in the contributing guidelines but there is no similar tool suggested for JS.

Reply:
No, but it's inconsistent with the rest.